src/ext/conversion/__test__/import-properties.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { Align, ColumnWidthType, ExportFormat } from '../../../types';
+import importProperties, { getColumnInfo, getMultiColumnInfo } from '../import-properties';
+
+describe('importProperties', () => {
+  const qDimensions = [
+    {
+      qDef: {
+        textAlign: {
+          auto: true,
+          align: 'left' as Align,
+        },
+        columnWidth: {
+          type: 'auto' as ColumnWidthType,
+          pixels: 200,
+          percentage: 20,
+        },
+      },
+      qAttributeExpressions: [],
+    },
+  ];
+  const qMeasures = [
+    {
+      qDef: {
+        qDef: '',
+        textAlign: {
+          auto: true,
+          align: 'left' as Align,
+        },
+        columnWidth: {
+          type: 'auto' as ColumnWidthType,
+          pixels: 200,
+          percentage: 20,
+        },
+      },
+      qAttributeExpressions: [],
+    },
+  ];
```

Comment:
the importProperties runs when you convert TO a sn-table right? then why would there be a `columnWidth` property on the qMeasures/qDimensions?

Reply:
It would just leave the column with no width info. Which is fine, since it is optional